docs/machine-learning/reference/ml-net-cli-reference.md

```diff
@@ -32,75 +32,136 @@ The third asset, the training code, shows you what ML.NET API code was used by t
 
 ```console
-mlnet auto-train
+mlnet classification
+
+--dataset <path> (REQUIRED)
 
---task | --mltask | -T <value>
+--label-col <col> (REQUIRED)
 
---dataset | -d <value>
+--cache <option>
 
-[
- [--validation-dataset | -v <value>]
-  --test-dataset | -t <value>
-]
+--has-header (Default: true)
 
---label-column-name | -n <value>
-|
---label-column-index | -i <value>
+--ignore-cols <cols>
 
-[--ignore-columns | -I <value>]
+--log-file-path <path>
 
-[--has-header | -h <value>]
+--name <name>
 
-[--max-exploration-time | -x <value>]
+-o, --output <path>
 
-[--verbosity | -V <value>]
+--test-dataset <path>
 
-[--cache | -c <value>]
+--train-time <time> (Default: 30 minutes, in seconds)
 
-[--name | -N <value>]
+--validation-dataset <path>
 
-[--output-path | -o <value>]
+-v, --verbosity <v>
 
-[--help | -h]
+-?, -h, --help
 
 ```
 
-Invalid input options cause the CLI tool to emit a list of valid inputs and an error message.
+### Regression options
+
+Running `mlnet regression` will train a regression model. Choose this command if you want an ML Model to predict a numeric value (e.g. price prediction).
+
+```console
+mlnet classification
+
+--dataset <path> (REQUIRED)
+
+--label-col <col> (REQUIRED)
+
+--cache <option>
+
+--has-header (Default: true)
+
+--ignore-cols <cols>
 
-## Task
+--log-file-path <path>
 
-`--task | --mltask | -T` (string)
+--name <name>
+
+-o, --output <path>
+
+--test-dataset <path>
+
+--train-time <time> (Default: 30 minutes, in seconds)
+
+--validation-dataset <path>
+
+-v, --verbosity <v>
+
+-?, -h, --help
+
+```
+
+### Recommendation options
+
+Running `mlnet recommendation` will train a recommendation model.  Choose this command if you want an ML Model to recommend items to users based on ratings (e.g. product recommendation).
+
+```console
+mlnet classification
 
-A single string providing the ML problem to solve. For instance, any of the following tasks (The CLI will eventually support all tasks supported in AutoML):
+--dataset <path> (REQUIRED)
 
-- `regression` - Choose if the ML Model will be used to predict a numeric value
-- `binary-classification` - Choose if the ML Model result has two possible categorical boolean values (0 or 1).
-- `multiclass-classification` - Choose if the ML Model result has multiple categorical possible values.
+--item-col <col> (REQUIRED)
 
-Only one ML task should be provided in this argument.
+--rating-col <col> (REQUIRED)
+
+--user-col <col> (REQUIRED)
+
+--cache <option>
+
+--has-header (Default: true)
+
+--log-file-path <path>
+
+--name <name>
+
+-o, --output <path>
+
+--test-dataset <path>
+
+--train-time <time> (Default: 30 minutes, in seconds)
+
+--validation-dataset <path>
+
+-v, --verbosity <v>
+
+-?, -h, --help
+
+```
+
+Invalid input options cause the CLI tool to emit a list of valid inputs and an error message.
```

Comment:
I would move this section just before the "Classification options" heading since it belongs to the entire section, not just the recommendation options.